README.adoc

```diff
@@ -69,26 +69,36 @@ Contributing
 
 Thank you for contributing to this repo. Please adhere to the following guidelines when creating a pull request:
 
-. Make your pull request against the _master_ branch (not develio) if you are contributing to the most recent release (currently
+. Make your pull request against the _master_ branch (not develop) if you are contributing to the most recent release (currently
   Cloud 10). The _master_ branch is protected and cannot be merged without review.
 
-. If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also 
+. If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also
   protected and cannot be merged without review.
 
 . Your contributions can be in ASCIIdoc `.adoc` or DocBook XML `.xml`.
 
+. Commit messages must conform to the following:
+.. Line lengths must be 80 characters or less.
+.. Commit body content is required.
+.. The subject line cannot start with "SOC".
```

Comment:
This is definitely in the regex but fwiw, I have no clue how this rule came to be.